oak-api/src/main/java/org/apache/jackrabbit/oak/api/jmx/QueryEngineSettingsMBean.java

```diff
@@ -139,5 +139,14 @@ void setQueryValidatorPattern(
 
     @Description("Get the query validator data as a JSON string.")
     String getQueryValidatorJson();
-
+    
+    
+//    @Description("Set or remove Java Package Name to ignore in Call Trace analysis")
```

Comment:
why do you comment this?
if i look at other methods, the '@Description' tag is above the method with some human readable information.

Reply:
Fixed